.github/workflows/compute.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           node-version: "20.10.0"
 
       - name: Install dependencies
-        run: yarn
+        run: npm install -g bun && bun install
```

Comment:
This can directly be used within https://bun.sh/guides/runtime/cicd
However I don't see where `bun` is used?